docs/get-started/tools/shimmer-bridge.md

```diff
@@ -0,0 +1,350 @@
+# ShimmerBridge User Guide
+
+## Introduction
+
+[ShimnmerBridge](https://shimmerbridge.org/) is a web application that allows you to transfer wrapped tokens between different networks by interacting with the decentralized smart contracts of [LayerZero](https://layerzero.network/). ShimmerBridge supports the **Ethereum, Avalanche C-Chain, Polygon, Arbitrum One, Optimism, Base, Fantom, and Binance Smart Chain** 
+networks. You can transfer several wrapped tokens, including **ETH, USDT, USDC, WBTC, AVAX, MATIC, FTM, and BNB.** 
+
+This guide explains how to bridge tokens with ShimmerBridge, including selecting the source chain, choosing the token to transfer, entering the number of tokens, and transferring the tokens. It also covers troubleshooting issues that may arise during the transfer process.
+
+The token addresses on each network are listed [at the end of this guide](#shimmerbridge-token-addresses) for your reference.
+
+## Bridge Assets with ShimmerBridge
+
+### 1. Accept the Terms and Conditions
+
+Go to **https://shimmerbridge.org/terms** and read the terms and conditions carefully before using ShimmerBridge. Once you understand the terms and conditions, enter the ShimmerBridge User Interface and connect your Web3 Wallet.
+
+
+![Layer Zero UI1](/img/introduction/Layer_Zero_Ui.png)
```

Comment:
Do you want to keep both images?

Then do the following and change the image names
```suggestion
<ThemedImage
  alt="Layer Zero UI"
  sources={{
    light: useBaseUrl('/img/introduction/Layer_Zero_Ui_light.png'),
    dark: useBaseUrl('/img/introduction/Layer_Zero_Ui_dark.png'),
  }}
/>;
```

Otherwise delete the duplicate image